src/jaspResults.cpp

```diff
@@ -127,7 +133,7 @@ std::string jaspResults::getStatus()
 void jaspResults::prepareForWriting()
 {
 	//Remove the seal if it is there or not doesnt matter
-	boost::nowide::remove((_writeSealRoot + _writeSealRelative).c_str());
```

Comment:
This could break windows when the file is stored in the appdata of a user with non-ascii characters in their username on a system not in a latin-locale...

So this should be changed back into `boost::nowide`.